fastlane/Fastfile

```diff
@@ -121,6 +121,21 @@ lane :test do |options|
     )
 end
 
+desc "Runs ui tests in Debug config"
+lane :test_ui do |options|
+
+  setCIEnvironmentVariable("../Tests/StreamChatUITests/StreamChatUITestPlan.xctestplan")
```

Comment:
You suggested that this should be run only if Fastlane's `is_ci` is `true` and I think it's a great idea. We can maybe add the check inside `setCIEnvironmentVariable`?